upload-apk.sh

```diff
@@ -11,22 +11,21 @@ git config --global user.email "noreply@travis.com"
 git config --global user.name "Travis-CI"
   
 git clone --quiet --branch=apk https://the-dagger:$GITHUB_API_KEY@github.com/fossasia/open-event-android  apk > /dev/null
-cd apk
```

Comment:
You are cloning the app branch in a folder named app, so you need to cd to that folder too.